x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/components/deduplication_panel/deduplication_panel.scss

```diff
@@ -0,0 +1,14 @@
+.deduplicationPanel {
+  .selectableWrapper {
+    padding: 1rem;
+    border-radius: .675rem;
+    border: 1px solid $euiColorLightestShade;
+  }
+
+  .showAllFieldsPopoverToggle {
+    .euiButtonEmpty__content {
+      padding-left: 12px;
+      padding-right: 12px;
```

Comment:
We should be using the EUI size variables here.
```
$euiSize  = 16rem
$euiSizeM = 12px
$euiBorderThin
```

Reply:
I've updated these in 4dcfb27